### PR TITLE
Change DERP protocol identifier and STUN software.

### PR DESCRIPTION
**Pull request by tomaszklak**
This was already tested manually with `tcl`i as described in the ticket. Additionally `STUN` module already includes appropriate tests so only `DERP` test is added.

### :ballot_box_with_check: Definition of Done checklist
- [x] Commit history is clean ([requirements](../blob/main/docs/git_commit_messages_requirements.md))
- [x] README.md is updated
- [x] changelog.md is updated
- [x] Functionality is covered by unit or integration tests

### CODE REVIEW

File: crates/telio-relay/src/derp/http.rs [hunk 1]
```diff
@@ -259,7 +259,7 @@ async fn connect_http<R: AsyncRead + Unpin, W: AsyncWrite + Unpin>(
                 "GET /derp HTTP/1.1\r\n\
                 Host: {host}\r\n\
                 Connection: Upgrade\r\n\
-                Upgrade: DERP\r\n\
+                Upgrade: WebSocket\r\n\
```
Comment (packgron) [inline crates/telio-relay/src/derp/http.rs:262]: Does derp servers already support basic websockets ?

**Comment by lcruz99** [comment]
nat-lab job is failing due to flaky test: `test_dns_duplicate_requests_on_multiple_forward_servers`

File: crates/telio-relay/src/derp/http.rs (hunk 1 above)
Comment (lcruz99) [reply crates/telio-relay/src/derp/http.rs:262]: nat-lab tests are passing at least

**Review by packgron — APPROVED** [review]
+1

File: crates/telio-relay/src/derp/http.rs (hunk 1 above)
Comment (tomaszklak) [reply crates/telio-relay/src/derp/http.rs:262]: I've tested it with tcli and was able to establish derp connection and I think that during refinement @Jauler checked the derp server implementation also.

**Review by packgron — APPROVED** [review]

**Pull request merged**